h/migrations/env.py

```diff
@@ -27,15 +28,10 @@
 # ... etc.
 
 
-def get_database_url(hint):
+def get_database_url():
     if 'DATABASE_URL' in os.environ:
-        urlparse.uses_netloc.append("postgres")
-        urlparse.uses_netloc.append("sqlite")
-        url = urlparse.urlparse(os.environ["DATABASE_URL"])
-        if url.scheme == 'postgres':
```

Comment:
ahh, I think this is my fault. It was fixed in the config.py (to convert to a list) but not here.